packages/app/src/features/savings/components/accounts-navigation/AccountsNavigation.tsx

```diff
@@ -0,0 +1,68 @@
+import { Token } from '@/domain/types/Token'
+import { TokenIcon } from '@/ui/atoms/token-icon/TokenIcon'
+import { cn } from '@/ui/utils/style'
+import { NormalizedUnitNumber } from '@marsfoundation/common-universal'
+
+interface AccountNavigationItem {
+  token: Token
+  deposited: NormalizedUnitNumber
+  active: boolean
+  onClick: () => void
+}
+
+export interface AccountsNavigationProps {
+  accounts: AccountNavigationItem[]
+}
+
+export function AccountsNavigation({ accounts }: AccountsNavigationProps) {
+  return (
+    <div className="flex gap-2 sm:flex-col">
+      {accounts.map((account) => (
+        <NavigationItem key={account.token.symbol} {...account} />
+      ))}
+    </div>
+  )
+}
+
+function NavigationItem({ token, deposited, active, onClick }: AccountNavigationItem) {
+  const transitionStyles = 'transition-all duration-200 ease-in-out'
```

Comment:
```suggestion
  const transitionStyles = cn('transition-all duration-200 ease-in-out')
```
For prompts from IDE